common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -90,7 +90,7 @@ public enum Feature {
 
         FIPS("FIPS 140-2 mode", Type.DISABLED_BY_DEFAULT),
 
-        DPOP("OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer", Type.PREVIEW);
+        LIGHT_WEIGHT_ACCESS_TOKEN("Light weight access token", Type.PREVIEW);
```

Comment:
```suggestion
        DPOP("OAuth 2.0 Demonstrating Proof-of-Possession at the Application Layer", Type.PREVIEW),
        LIGHT_WEIGHT_ACCESS_TOKEN("Light weight access token", Type.PREVIEW);
```

DPOP is used in Keycloak code. If you see this PR does not build.

Reply:
Thank you for pointing it out.
I fixed it.

Reply:
Yes